upgrade/step6/index.md

```diff
@@ -0,0 +1,317 @@
+
+Tensioning screws (M5x25mm button head cap screw) are used at one end of each length of belt to help keep your belts tight. Place the M5x25mm button head cap screw (tensioning screw) through the hole in the small bend of the belt clip. Place the M5x8mm button head cap screw through the remaining hole and lightly tighten the clip down to the pre-assembly insertion nut. Place the tensioning screw through the hole in the back end plate and hand tighten the M5 nylon locking nut. 
+
+<img src="photos/IMG_6166.JPG">
+
+On the opposite end, without the tensioning screw, place the M5x8mm button head cap screw through the hole on the belt clip and lightly tighten the clip down to the pre-assembly insertion nut.
+
+<img src="photos/IMG_6165.JPG">
+
+Your Y-axis stepper motor and belt update is now complete. 
+
+<h3> Attach z-axis stepper motor and belt </h3>
+
+The x-axis belt has a few extra steps. If you did not remove your x-axis belt previously, start by removing by loosening the belt clips at each end and removing the hardware. 
+
+<img src="photos/IMG_6168.JPG">
+<img src="photos/IMG_6171.JPG">
+
+Next, loosen and remove the x-axis stepper motor. Lastly, you’ll remove the existing idlers.<br> 
+<strong>We will be reusing the hardware, so take care to keep track of it.</strong>
+
+<img src="photos/IMG_6065.JPG">
+<img src="photos/IMG_6069.JPG">
+<img src="photos/IMG_6081.JPG">
+
+After the x-axis motor, idlers and belt are removed, disconnect the z-stepper motor, homing switch wires and the drag chain bracket. You'll also need to carefully disconnect the z-probe from the z-probe relocation bracket. 
+
+<img src="photos/IMG_6177.JPG">
+<img src="photos/IMG_6181.JPG">
+<img src="photos/IMG_6186.JPG">
+
+Loosen the top 2 screws that hold the right side plate to the wide makerslide. Before removing the screws, use a box or prop to hold the makerslide and drag chain rail in place to keep it from falling and bending. Be careful as the drag chain arm brackets are fragile and can snap off.  
+
+<img src="photos/IMG_6189.JPG">
+
+
+Once the wide makerslide and drag chain rail are propped up, completely remove the screws and slide the Y2 side plate forward/backward out of the way. 
+
+<img src="photos/IMG_6195.JPG">
+<img src="photos/IMG_6203.JPG">
+
+Carefuly slide the x-carriage off of the wide makerlide. You may need to turn the eccentric spacer to the open position to allow the x-carraige to move easily. 
+
+<img src="photos/IMG_6208.JPG">
+
+Using the hardware set aside earlier, install the 9mmm belt idlers. The order is as follows: screw through chassis, spacer, belt idler, two M5 wasters and locknut. Make sure the inset part is facing out/away from the screw head.
+
+<img src="photos/IMG_6212.JPG">
+<img src="photos/IMG_6240.JPG">
+<img src="photos/IMG_6243.JPG">
+
+This part can be tricky. You may try using a bit of tape at the end of the wrench to help re-attach the nut. 
```

Comment:
I suggest "This step requires precision and patience" rather than "this part can be tricky"